examples/r-shiny/README.md

```diff
@@ -0,0 +1,125 @@
+```
+
+Now let us create another R script called `app.R`. This script will contain Shiny Dashboard. We have created a simple dashboard which has a slider and a plot.
+User will select a number as single input from slider This number represents value of bins for histogram plot  In the following code in first line we have loaded the ggplot2 and shiny packages. 
+Now we will read house price data in variable data.
+
+In UI function we will create appearance of our dashboard. Title of our dashboard "Housing Data" is added to title panel.
+We then give structure to our app using sidebarLayout function. sidebarLayout contains sidebarPanel and mainPanel. In slider input we are setting number of bins ranging from 1 to 30.
+Since we want our output to show histogram plot, we will use function plotOutput inside mainPanel. 
+plotOutput displays outputs returned from render functions (renderPlot in our case). 
+
+In server function we have logic to create histogram as per selected number of bins from slider. Function renderPlot will return output to function plotOutput. 
+In our case renderPlot fetches qplot object and stores it in variable `distPlot`. From housing dataset we first extract sale prices of houses. Then generate bins as per the user input and then draw histogram based on bin number using qplot function. 
+
+In the last line we are call shinyApp and pass in UI and server variables.
+
+```R
+library(ggplot2)
+library(shiny)
+
+data<-read.csv("https://saturn-public-data.s3.us-east-2.amazonaws.com/examples/dashboard/housePriceData.csv")
+ui <- fluidPage(
+    titlePanel("Housing Data"),
+    sidebarLayout(
+        sidebarPanel(
+            sliderInput("bins",
+                        "Number of bins:",
+                        min = 1,
+                        max = 30,
+                        value = 15)
+        ),
+        mainPanel(
+           plotOutput("distPlot")
+        )
+    )
+)
+server <- function(input, output) {
+      output$distPlot <- renderPlot({
+        x<- data$SalePrice
+        bins <- seq(min(x), max(x), length.out = input$bins + 1)
+        qplot(x, geom="histogram",breaks =bins,   
+              main = "Histogram for Sale Prices for houses", 
+              xlab = "Sale Price",  
+              fill=I("Orange"), 
+              col=I("darkgray"))        
+    })
+}
+shinyApp(ui = ui, server = server)
+
+```
+Now create another R script which will run above piece of code. Let's name this as `run-app.R`. Start the server using shiny object. Since the host listens on 0.0.0.0, it will be reachable on an appropriate interface address to the connection. The API must use port 8000 for Saturn Cloud to be able to direct traffic to it.
+```R
+shiny::runApp(host="0.0.0.0", port=8000, launch.browser=FALSE)
+```
+
+## Deployment
+
+Save all three files (`setup.R`, `app.R`, `run-app.R`) in a directory inside GitHub repo. Now click **New Deployment**. It can be found on top right side of the resource page. 
```

Comment:
Moved the setting up git stuff from below

```suggestion
Save all three files (`setup.R`, `app.R`, `run-app.R`) in a directory inside GitHub repo. You'll need to connect [git on Saturn Cloud](https://saturncloud.io/docs/using-saturn-cloud/gitrepo/) to your git repo host. Now click **New Deployment**. It can be found on top right side of the resource page. 
```